qiime2/plugins.py

```diff
@@ -188,12 +188,12 @@ def init_format(self, name, factory, ext=None):
         return variable
 
     def import_from_format(self, name, semantic_type,
-                           fmt_variable, view_type=None):
+                           variable, view_type=None):
         variable = super().import_from_format(
-            name, semantic_type, fmt_variable, view_type=view_type)
+            name, semantic_type, variable, view_type=view_type)
 
         interface_name = variable.to_interface_name()
-        import_fp = fmt_variable.to_interface_name()
+        import_fp = variable.to_interface_name()
```

Comment:
Namespace collision here same as in q2cli. (Use `var` for the super'ed return result to avoid)

Reply:
This is why the sig name was `fmt_variable`, fwiw.

Reply:
Not that that's a good reason or anything, this is just for context. I could've just as easily picked a different name for the `super()` result, but such is life.